oar/cli/cmd_drop_bugs.py

```diff
@@ -34,6 +35,24 @@ def drop_bugs(ctx):
             NotificationManager(cs).share_dropped_and_must_verify_bugs(
                 dropped_bugs, must_verify_bugs
             )
+            requested_doc_ads=[]
+            requested_prodsec_ads=[]
+            if len(approved_doc_ads):
+                for ad in approved_doc_ads:
+                    ad.refresh()
+                    if not ad.is_doc_approved():
+                        if not ad.is_doc_requested():
+                            ad.request_doc_approved()
+                            requested_doc_ads.append(ad)
+            if len(approved_prodsec_ads):
+                for ad in approved_prodsec_ads:
+                    ad.refresh()
+                    if not ad.is_prodsec_approved():
+                        if ad.is_prodsec_requested() == 'null': 
+                            ad.request_prodsec_approved()
+                            requested_prodsec_ads.append(ad)
+            logger.info(requested_doc_ads,requested_prodsec_ads)
```

Comment:
suggest to add some description to make this log entry more meaningful. what are the output ids